Autopilot/Inc/IMU.hpp

```diff
@@ -18,7 +18,7 @@ struct IMUData_t {
     float gyrx, gyry, gyrz; 
 
     bool isDataNew; 
-    int status; //TBD but probably 0 = SUCCESS, -1 = FAIL, 1 = BUSY 
+    int sensorStatus; //TBD but probably 0 = SUCCESS, -1 = FAIL, 1 = BUSY 
```

Comment:
Curious, could this be an enum instead so that we can add in more states if required and keep it clean?